app/src/main/java/org/sagebionetworks/research/mpower/medication/MedicationViewModel.kt

```diff
@@ -0,0 +1,109 @@
+package org.sagebionetworks.research.mpower.medication
+
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.MutableLiveData
+import android.arch.lifecycle.ViewModel
+import org.slf4j.LoggerFactory
+
+class MedicationViewModel : ViewModel() {
+
+    private val LOGGER = LoggerFactory.getLogger(
+            MedicationViewModel::class.java)
+
+    private val title = MutableLiveData<String>()
```

Comment:
There is a TrackingStep that we have been using as a model object for the data that comes from json. I need to look into the medication json and make the necessary changes, but we should eventually shoot for this view model to get some of it's data from that.